Gemfile

```diff
@@ -7,56 +7,33 @@ git_source(:github) do |repo_name|
   "https://github.com/#{repo_name}.git"
 end
 
-
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
```

Comment:
OK, I've added back in comments, including additional comments to make it consistent.